src/arrays.jl

```diff
@@ -135,19 +136,23 @@ function Base.Broadcast.broadcasted(
     u::UniFinArr{S,V,R,P,N},
     cv::CategoricalValue) where {S,V,R,P,N}
 
-    cv in classes(u) || throw(err_missing_class(cv))
+    # we assume that we compare categorical values by their unwrapped value
+    # and pick the index of this value from classes(u)
+    cv_loc = findfirst(==(cv), classes(u))
```

Comment:
@bkamins. I know we have this merged. But can't we use 
```
cv_loc = findfirst(==(cv), get(CategoricalArrays.pool(classes(u)), cv))
```

Reply:
A very good point. It will be indeed much better. This is also in:
https://github.com/JuliaAI/CategoricalDistributions.jl/pull/40/files#diff-d2b76a04a67d354b16ee7ce57fc14746942b325d90fdf80764304284fac50bc0R169